2014-10-21 Kai Pastor (OpenOrienteering)
- Updated for Qt-5.4.0-beta

2014-09-19 Kai Pastor (OpenOrienteering)
- Never fail on missing CMake tests, as we remove all tests from source dir.

diff -up Qt5/qtbase/mkspecs/features/create_cmake.prf.orig Qt5/qtbase/mkspecs/features/create_cmake.prf
--- Qt5/qtbase/mkspecs/features/create_cmake.prf.orig
+++ Qt5/qtbase/mkspecs/features/create_cmake.prf
@@ -305,7 +305,8 @@
 # with 'CONFIG -= create_cmake'
 !equals(CMAKE_MODULE_TESTS, -) {
     isEmpty(CMAKE_MODULE_TESTS): CMAKE_MODULE_TESTS = $$MODULE_BASE_INDIR/tests/auto/cmake
-    !exists($$CMAKE_MODULE_TESTS): \
-        error("Missing CMake tests. Either create tests in tests/auto/cmake," \
-              "or disable cmake config file creation with CONFIG-=create_cmake.")
+    # Test dirs were removed in OpenOrienteering build.
+    #!exists($$CMAKE_MODULE_TESTS): \
+    #    error("Missing CMake tests. Either create tests in tests/auto/cmake," \
+    #          "or disable cmake config file creation with CONFIG-=create_cmake.")
 }
